Exception/MissingAccountException.php

```diff
@@ -29,16 +29,29 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
  * @author Nosto Solutions Ltd <contact@nosto.com>
- * @copyright 2020 Nosto Solutions Ltd
+ * @copyright 2021 Nosto Solutions Ltd
  * @license http://opensource.org/licenses/BSD-3-Clause BSD 3-Clause
  *
  */
 
 namespace Nosto\Cmp\Exception;
 
-use Nosto\NostoException;
+use Exception;
+use Throwable;
 
-class MissingCookieException extends NostoException
+class MissingAccountException extends Exception
 {
-
+    /**
+     * MissingAccountException constructor.
+     * @param string $message
+     * @param int $code
+     * @param Throwable|null $previous
+     */
+    public function __construct(
+        $message = 'Account cannot be null',
```

Comment:
The message is not overridable and should be a constant. It should be removed along with the `code` and `previous` parameters